src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -269,11 +328,115 @@ const handleOpenAuth = (mediator, artist) => {
   })
 }
 
+const RoundedFlex = styled(Flex)`
+  border-radius: 100px;
+`
+
+const renderAuctionHighlight = artist => {
+  if (
+    !artist.auctionResultsConnection ||
+    artist.auctionResultsConnection.edges.length < 1
+  ) {
+    return null
+  }
+  const topAuctionResult =
+    artist.auctionResultsConnection.edges[0].node.price_realized.display
+
+  return (
+    <RoundedFlex
+      background={color("black5")}
+      width="auto"
+      py={5}
+      px={10}
+      mr={5}
+    >
+      <AuctionIcon pr={5} />
+      <Sans size="2">{topAuctionResult} Auction Record</Sans>
+    </RoundedFlex>
+  )
+}
+
+const CATEGORIES = {
+  "blue-chip": "Blue chip",
+  "top-established": "Established",
+  "top-emerging": "Emerging",
+}
+
+const ICON_MAPPING = {
+  BLUE_CHIP: BlueChipIcon,
```

Comment:
Feels odd to use this format instead of just `"blue-chip"`, etc. as keys here.

Reply:
this actually exists somewhere else. Is it better to import it and leave the mapping as is or change it like you suggest?